folly/Traits.h

```diff
@@ -384,10 +384,24 @@ struct Bools {
   }
 };
 
+template <bool... Bs>
+struct AllFalse : std::is_same<Bools<Bs...>, Bools<(Bs && false)...>> {};
```

Comment:
I would make these aliases and internal implementation details, but my preference is to just leave them out entirely.